index.html

```diff
@@ -0,0 +1,146 @@
+              <li><a href="">English</a></li>
+              <li><a href=""> My Page</a></li>
+          </ul>
+      </div>
+  </div>
+  <div class="desktop-navbar">
+      <div class="logo">
+          <img src="./images/riana.png" alt="logo-image">
+      </div>
+      <div class="desktop-nav-list">
+          <ul>
+              <li><a href="about.html">About</a></li>
+              <li><a href="#team-id">program</a></li>
+              <li><a href="#team-id"> Sponsor</a></li>
+              <li><a href="#team-id"> News</a></li>
+          </ul>
+      </div>
+  </div>
+  </nav>
+  <div class="welcome">
+    <h3 class="text">"Hello how are you!"</h3>
+    <h1 class="text">Concert King's Jimin</h1>
+  </div>
+  <div class="welcome-description">
+    <p class="welcome-text">
+       Stage King' Jimin wows fans and netizens with his exhilarating dance,
+       vocals and visuals after BTS's Busan concert
+    </p>
+  </div>
+  <div class="event-date">
+    <h2 class="fan-text">Be fan now</h2>
+    <p class="fan-p">@ BTS, The world rocking <strong>boys</strong></p>
+  </div>
+</div>
+</section>
+    </header>
+        <section class="program">
+          <h2 id="services-title">Main songs on stage</h2>
+          <div id="redline"></div>
+          <div class="service-cards">
+          <div class="card">
+              <img src="./images/permission.png" class="card-images" alt="">
+              <h3 id="program-title"> Permission </h3>
+              <p> you can watch the recorded concert on bts main channel.</p>
+          </div>
+          <div class="card">
+              <img src="./images/fire.png" class="card-images" alt="">
+              <h3 id="program-title"> fire</h3>
+              <p>you can watch the recorded concert on bts main channel.</p>
+          </div>
+          <div class="card">
+              <img src="./images/dynimate.png" class="card-images" alt="">
+              <h3 id="program-title"> Dynamite</h3>
+              <p>you can watch the recorded concert on bts main channel.</p>
+          </div>
+          <div class="card">
+              <img src="./images/butter.png" class="card-images" alt="">
+              <h3 id="program-title"> Butter </h3>
+              <p> you can watch the recorded concert on bts main channel.</p>
+          </div>
+          <div class="card">
+              <img src="./images/fake love .png" class="card-images" alt="">
+              <h3 id="program-title"> Fake love</h3>
+              <p>  you can watch the recorded concert on bts main channel.</p>
+          </div>
+      </div>
+      <div id="view-prog-div">
+          <a id="view-program" href=""> To View full concert &gt;</a>
+            <a id="view-program" href="#" class="text-white ">
+              Join BTS main channel
+            </a>
+      </div>
+      </section>
+      <section id="team-id" class="team-members">
+        <h2 id="team-title">Featured speakers</h2>
```

Comment:
- Please re-position here too and make sure the button section contains a section header and a **divider**.
Your Design:
![Capture](https://user-images.githubusercontent.com/63972670/198002293-2cebd6ee-fb17-4522-a8de-90c99d1e35c3.PNG)
Original Design:
![Capture](https://user-images.githubusercontent.com/63972670/198002533-c06c1027-b78c-4080-bbdf-a9cdc8034a7f.PNG)